webapp/packages/core-blocks/src/ObjectPropertyInfo/ObjectPropertyInfoForm/RenderField.tsx

```diff
@@ -229,11 +230,21 @@ export const RenderField = observer<RenderFieldProps>(function RenderField({
     );
   }
 
+  let type: HTMLInputTypeAttribute = 'text';
+
+  if (controlType === 'input:number') {
+    type = 'number';
+  }
+
+  if (password) {
+    type = 'password';
+  }
+
```

Comment:
we have controlType (`getObjectPropertyType`).
Now we also have different types of data, so we probably need to create a function like `getObjectPropertyType`.
Let's call it `getObjectPropertyValueType`. This new function will return `'password' | 'text' | 'number'` (and we can extend it later without changing the logic in `RenderField` it also will guarantee that all control types will work the same way